xbmc/filesystem/CurlFile.cpp

```diff
@@ -634,6 +636,16 @@ void CCurlFile::SetCommonOptions(CReadState* state, bool failOnError /* = true *
   else
     // enable HTTP2 support. default: CURL_HTTP_VERSION_1_1. Curl >= 7.62.0 defaults to CURL_HTTP_VERSION_2TLS
     g_curlInterface.easy_setopt(h, CURLOPT_HTTP_VERSION, CURL_HTTP_VERSION_2_0);
+
+  const CURLcode altSvcCacheResult = g_curlInterface.easy_setopt(h, CURLOPT_ALTSVC, m_curlAltSvcCachePath.c_str());
```

Comment:
> The `ColumnLimit` in `.clang-format` is set to `100` which defines line length [...]

https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#3-formatting

Reply:
Thanks, I've added new lines to meet the line length constraint.